### ISSUE TITLE
WebServiceEntity and SyncUrl seem to be identical

### ISSUE THREAD

**Issue by alxndrsn** [ushahidi/SMSSync#431]
There are two classes which look identical: [`WebServiceEntity`](https://github.com/ushahidi/SMSSync/blob/master/smssync/src/main/java/org/addhen/smssync/domain/entity/WebServiceEntity.java) and [`SyncUrl`](https://github.com/ushahidi/SMSSync/blob/master/smssync/src/main/java/org/addhen/smssync/data/entity/SyncUrl.java).  I suspect one of them is outdated, but they're both used in various different places in the codebase, so it's not clear if that's actually the case.

**Comment by eyedol** [comment]
@alxndrsn This is a technical debt. The database library uses the class name as the table name. I changed the class names when I was refactoring the code base. Later I discovered I had to maintain the old class name, `SyncUrl` if I don't want to update the db. This will require writing more code. It needs revisiting. Perhaps for now note a comment in `SyncUrl` why it's name is so?

**Comment by alxndrsn** [comment]
If `SyncUrl` has to stay, would it be possible to remove `WebServiceEntity`?

**Comment by eyedol** [comment]
We can do that but it'll change the nomenclature all together. You can go for it though.

**Comment by alxndrsn** [comment]
Looks like we could change the database table name in [`BaseDatabaseHelper.onUpgrade()`](https://github.com/ushahidi/SMSSync/blob/a1f2224bbb4488a9772eb35a1d8648e0dc350a59/smssync/src/main/java/org/addhen/smssync/data/database/BaseDatabaseHelper.java#L96), and therefore get rid of `SyncUrl`.

**Comment by eyedol** [comment]
@alxndrsn Yeah you can. You've to do the data migration. There is no decent migration tool for Android as far as I know. Mostly I write them manually. Say if you upgrade from the app from x.x.y to x.x.z, all the saved data has to move with it. Not sure if there is a `sql` query to rename a table without having to create temp tables and move data around.

**Comment by alxndrsn** [comment]
Something like:
```SQL
ALTER TABLE SyncUrl RENAME TO WebSericeEntity
```
should do it

**Comment by eyedol** [comment]
That should probably do the trick
